### PR TITLE
feat!: run the generator 2020/12

### PR DESCRIPTION
**Pull request by grant**
Runs the generator. Looks like there are a few updates:

- stricter, more accurate types
- non-nullable values
- fixed number types (used to be number | string, like `diskSizeGb`)
- better enum support
- updated copyright header

This is a breaking change.

### CODE REVIEW

File: cloud/audit/v1/LogEntryData.ts [hunk 1]
```diff
@@ -498,7 +495,7 @@ export interface DestinationAttributes {
     /**
      * The network port of the peer.
      */
-    port?: number | string;
+    port?: number;
```
Comment (grayside) [inline cloud/audit/v1/LogEntryData.ts:498]: Does this allow the generated library to surface errors more easily, or is this adding new errors where previously the underlying data structures/APIs would accept a string?

**Review by grayside — CHANGES_REQUESTED** [review]
Making this a blocking review to confirm we don't need a major version update.

File: cloud/audit/v1/LogEntryData.ts (hunk 1 above)
Comment (grant) [reply cloud/audit/v1/LogEntryData.ts:498]: This gives a narrower definition of the properties so a user does not need to check if the data is a string or number.

The port is always a number from the API for example.

**Review by grayside — APPROVED** [review]
Approving given my understanding this will lead to tagging a v4 of the library.

**Pull request merged**